tools/uswds-component-library-figma.md

```diff
@@ -29,26 +31,27 @@ You've successfully duplicated the library! Now you can publish this file and ap
 
 ![GIF demonstrating how to navigate through Figma's community page to duplicate the U.S. Web Design System library created by Truss.]({{ site.url }}/{{site.baseurl}}/assets/img/uswds-component-library-publishing-file.gif)
 
-## Start crafting designs 
+## Crafting designs 
 
 Not sure where to start or need inspiration? 
 
-We've gone ahead and created some page templates using the USWDS components library. Check them out [here](#)!
-<!--need to link to USWDS page templates later-->
+We've gone ahead and created some [page templates](https://bixal.github.io/uswds-page-templates/) using the USWDS components library. You can view how they respond in the web or view their Figma files to dive a little deeper.
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="800" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FQVPduB8h6DIENYULFVCism%2FBixal-USWDS-Page-Templates%3Fnode-id%3D1453%253A2736" allowfullscreen></iframe>
```

Comment:
I like the Figma embed! We'll need to make it responsive. There's a technique for doing this in our Video page template. But we can merge for now and capture that as a separate issue.